forge-gui/res/cardsfolder/upcoming/ominous_lockbox.txt

```diff
@@ -0,0 +1,10 @@
+Name:Ominous Lockbox
+ManaCost:1 U
+Types:Artifact Clue
+K:ETBReplacement:Other:ChooseNum
+SVar:ChooseNum:DB$ ChooseNumber | SecretlyChoose$ True | Min$ 1 | Max$ 20 | KeepSecret$ True | SpellDescription$ As CARDNAME enters the battlefield, secretly choose a number between 1 and 20.
```

Comment:
can you also rename this param to Secretly?
then they should finally all be consistent...

Reply:
I think ChooseType is already `Secretly$`